README.md

```diff
@@ -1,3 +1,5 @@
 # Demo Gradle Dependabot
 
 Demonstrates the use of Dependabot for Kotlin/Gradle projects through the Dependency Submission API.
+
```

Comment:
why do you have the whitespace here?

Reply:
well i added a new paragraph and md usually has a space between the paragraphs